pytrace-generator/main.py

```diff
@@ -16,7 +16,7 @@ def eprint(*args, **kwargs):
     print(*args, file=sys.stderr, **kwargs)
 
 type_name_regex = re.compile("<class '(?:__main__\\.)?(.*)'>")
-import_regex = re.compile(r"^(?:[^'\"]+\s)?import ")
+import_regex = re.compile(r"^(?:[^'\"]+\s)?import[\s*]")
```

Comment:
Because leaving out the whitespace character after the import (from what I tested) only works when a `*` is following. `importwypp` would not be valid, for example. If we don't care about that, we can then also omit the entire `[\s*]`. What do you prefer?

Reply:
ok, all good